data/json/npcs/talk_tags_follower.json

```diff
@@ -590,6 +590,12 @@
       "zero chance"
     ]
   },
+  {
+    "type": "snippet",
+    "category": "<bye>",
```

Comment:
There is already a similar `<end_talking_bye>` category, maybe use it? Or put this category inside a new one?

https://github.com/CleverRaven/Cataclysm-DDA/blob/052442d5c557865497c1aa72c18b56f76dce0f14/data/json/npcs/talk_tags_avatar.json#L59-L65

Reply:
yeah i just didn't know one exists already